content/plugins/fabric.md

```diff
@@ -238,3 +237,47 @@ Using a tasks file instead of a list of commands will allow you to use python co
 Using a list of commands might be a good solution for very simple cases in which you wouldn't want to maintain a tasks file.
 
 {{% /gsTip %}}
+
+
+# ctx for fabric-plugin
+
+Starting with Cloudify 3.4 and fabric-plugin 1.4, we now support using ctx in Python scripts executed by the fabric plugin on remote machines. Most of the functionality is similiar to how the script plugin exposes the ctx object.
```

Comment:
i don't think the word "we" belongs here. Perhaps: "Cloudify now supports", "The fabric plugin now supports", etc...